main/glossary/README.md

```diff
@@ -67,151 +102,362 @@ synchronously compared for structural equivalence.
 
+## Offer
+
+Offers are a structured way of describing user intent. In Zoe, an offer consists of a [proposal](#proposal) (the 
+rules under which the party wants to exercise the offer) and [payments](#payment) corresponding to what the proposal specifies as what the
+party will give if the offer is satisfied. The payments are automatically [escrowed](#escrow) by Zoe, and appropriately reallocated when
+the offer exits with either success or rejection. See [`E(Zoe).offer(invitation, proposal, paymentKeywordRecord)`](https://agoric.com/documentation/zoe/api/zoe.html#e-zoe-offer-invitation-proposal-paymentkeywordrecord).
+
+## Offer Safety
+
+Zoe guarantees offer safety. When a user makes an [offer](#offer) and it is [escrowed](#escrow) with Zoe, Zoe guarantees that 
+the user either gets what they said they wanted, or gets back (refunded) what they originally offered and escrowed.
+
 ## Payment
-Payments hold [amounts](#amounts) of certain assets 
-issued by [Mints](#mint). Specifically amounts that are *in transit* from one party to another. 
-Amounts from payments can be deposited in [purses](#purse), but otherwise, the entire amount is 
-available when the payment is transferred. Payments can be converted to [Purses](#purse). All contents 
-of a purse must be of the same [brand](#brand).
-
-See the [ERTP Guide's Payments section](/ertp/guide/purses-and-payments.md#payments)
-and the [ERTP API's Payments section](/ertp/api/payment.md#payment-getallegedbrand) 
-for more information.
-
-## Presence
-A local version of a remote object that serves as a proxy for the remote object. 
-If `obj` is a presence of a remote object, you can send messages to the remote object by using `E()` on `obj`. 
-See the [JavaScript Distributed Programming Guide](/distributed-programming.md) for more information. 
 
-## Purse
-Purses hold [amounts](#amounts) of certain [mint](#mint) issued assets. Specifically amounts that are _stationary_. Purses can transfer part of their held balance to a [payment](#payment), which is usually used to transfer value. A purse's contents are all of the same [brand](#brand).
+Payments hold assets issued by [Mints](#mint). Specifically assets intended for transfer 
+from one party to another. All assets of a payment must be of the same [brand](#brand).
+
+For more information, see the [ERTP Guide's Payments section](/ertp/guide/purses-and-payments.md#purses-and-payments)
+and the [ERTP API's Payments section](/ertp/api/payment.md).
 
-See the [ERTP Guide's Purses section](/ertp/guide/purses-and-payments.md#purses) and the
-[ERTP API's Purses section](/ertp/api/purse.md)
-for more information.
+## Payout
+The assets reallocated to a user when an [offer](#offer) exits, either successfully or not. If an offer isn't successful, a
+payout is a party's [escrowed](#escrow) assets. If an offer is successful, a party's payout is what they wanted, and possibly the 
+return of any escrowed assets in excess of the final cost. 
+
```

Comment:
This is very confused. There's no concept of offer success, and the payout does not depend on it. The payout is always what the seat's current allocation is, regardless of how the seat exited (seat.exit() or seat.fail() or an error in the offerHandler). If there was a previous reallocation, it will be different than what the user escrowed. If there is no reallocation before the seat exits, it will be the same as what they escrowed.